src/component/common/PermissionIconButton/PermissionIconButton.tsx

```diff
@@ -8,7 +8,7 @@ import {
 } from 'component/common/TooltipResolver/TooltipResolver';
 import { formatAccessText } from 'utils/formatAccessText';
 
-interface IPermissionIconButtonProps {
+interface IPermissionIconButtonProps extends IconButtonProps {
```

Comment:
I think we should avoid extending the built-in MUI prop types, as they're already quite complex.

Could add specific props like `edge?:` below, or use something like `tooltipProps?:` if we need many of them.

Reply:
Wouldn't you want the ability to use the properties of the icon button without having to manually update the theme every time you use one of the properties? Is there a significant drawback here?

Reply:
I guess it can become cluttered and confusing what props it actually takes. I will refactor.